reference/formats/expression_language.rst

```diff
@@ -489,6 +494,8 @@ The following table summarizes the operators and their associativity from the
 +-----------------------------------------------------------------+---------------+
 | ``or``, ``||``                                                  | left          |
 +-----------------------------------------------------------------+---------------+
+| ``xor``                                                         | left          |
```

Comment:
rereading the PR I guess that this line needs to be moved up as the table is order by operator precedence

Reply:
Does it look good now?

Reply:
yes, thanks 👍